tensorflow/core/ops/math_ops.cc

```diff
@@ -259,6 +259,12 @@ REGISTER_OP("ComplexAbs")
       .Attr("T: {bfloat16, half, float, double}") \
       .SetShapeFn(shape_inference::UnchangedShape)
 
+#define UNARY_REAL_6()                              \
```

Comment:
If it's only going to be used the once, just inline the registration.  `UNARY_REAL_6` isn't meaningful.

Though the round op could be supported - and is with XLA.  So we can add the kernel registrations if there is a real use-case, or tell the reporter to use XLA if they really require it.

Reply:
I tested this Op with jit_compile=True and found Invalid Argument error. [Gist](https://colab.sandbox.google.com/gist/SuryanarayanaY/07823f87bba4ade098de3bf6b8968e6d/65317_r1.ipynb).

`InvalidArgumentError: Operands to RoundToEven must be real-valued floating-point, but got C128
`

This seems XLA also not supports complex dtypes ?

Reply:
Weird, then why do they bother registering the op for complex?

We can at least add for the integers, but leave off the registrations for complex.

Reply:
The Macro Unary() registered for other Ops also like `raw_ops.Neg` which can accept complex dtypes also and works fine as well.Hence we can't remove complex dtypes from there.

So for Round Op we might need to have a separate Macro that can register int and floats. Hence I proposed a new macro for same

Reply:
You don't need a macro.  You can simply do:
```
RegisterOp("Round")
      .Input("x: T")
      .Output("y: T")
      .Attr("T: {bfloat16, half, float, double, int32, int64}")
      .SetShapeFn(shape_inference::UnchangedShape)
```
directly.

Reply:
Thanks.Registered the Op for all real dtypes directly as suggested.